docker/package/tezos_voting_wizard.py

```diff
@@ -0,0 +1,512 @@
+            ]
+        ),
+    )
+
+
+ballot_outcome_query = Step(
+    id="ballot_outcome",
+    prompt="Choose the outcome for your ballot.",
+    help="'yay' is for supporting the proposal, 'nay' is for rejecting the proposal, "
+    "'pass' is used to not influence a vote but still contribute to reaching a quorum.",
+    options=ballot_outcomes,
+    validator=Validator(
+        [required_field_validator, enum_range_validator(ballot_outcomes)]
+    ),
+)
+
+
+class Setup:
+    def __init__(self, config={}):
+        self.config = config
+
+    def query_step(self, step: Step):
+        validated = False
+        while not validated:
+            print(step.prompt)
+            step.pprint_options()
+            answer = input("> ").strip()
+
+            if answer.lower() in ["quit", "exit"]:
+                raise KeyboardInterrupt
+            elif answer.lower() in ["help", "?"]:
+                print(step.help)
+                print()
+            else:
+                if not answer and step.default is not None:
+                    answer = step.default
+
+                try:
+                    if step.validator is not None:
+                        answer = step.validator.validate(answer)
+                except ValueError as e:
+                    print(color("Validation error: " + str(e), color_red))
+                else:
+                    validated = True
+                    self.config[step.id] = answer
+
+    # Check whether the baker_alias account is set up to use ledger
+    def check_ledger_use(self):
+        baker_alias = self.config["baker_alias"]
+        address = get_proc_output(
+            f"sudo -u tezos {suppress_warning_text} tezos-client "
+            f"{self.config['tezos_client_options']} show address {baker_alias} --show-secret"
+        )
+        if address.returncode == 0:
+            value_regex = b"(?:" + ledger_regex + b")"
+            return bool(re.match(value_regex, address.stdout))
+
+    def check_baking_service(self):
+        try:
+            proc_call("systemctl is-active --quiet tezos-baking-mainnet.service")
+        except:
+            print("Looks like the tezos-baking-mainnet service isn't running.")
+            print("Please start the service or set up baking by running:")
+            print("tezos-setup-wizard")
+            sys.exit(1)
+
+    def check_data_correctness(self):
+        print("Baker data detected is as follows:")
+        print(f"Data directory: {self.config['client_data_dir']}")
+        print(f"Node RPC address: {self.config['node_rpc_addr']}")
+        print(f"Baker alias: {self.config['baker_alias']}")
+        if not yes_or_no("Does this look correct? (Y/n)", "yes"):
+            print("Try setting up baking again by running:")
+            print("tezos-setup-wizard")
+            sys.exit(1)
+
+    def fill_baking_config(self):
+        output = get_proc_output("systemctl show tezos-baking-mainnet.service").stdout
```

Comment:
Seems like we need #349 in order to test this wizard and a way to choose the network that is going to be used during the voting